packages/ui/src/tooltip/Tooltip.css

```diff
@@ -0,0 +1,24 @@
+.ui-tooltip-aria {
+  background-color: #252423;
+  color: #FFFFFF;
+  padding: var(--layout-margin-small, 12px);
+  margin-top: 10px;
+  max-width: 400px;
+  font-size: 14px;
+  border-radius: 4px;
+  background: var(--surface-color-brand-gray-110, #252423);
+  box-shadow: 0px 0.6px 1.8px 0px rgba(0, 0, 0, 0.10), 0px 3.2px 7.2px 0px rgba(0, 0, 0, 0.13);
```

Comment:
Same with this. This looks to be --surface-shadow-small. For reference, Zac tried to keep a lot of the theming in line with the BC Design Theme and the variables file is an export of their theme library variables.